text/0265-event-vision.md

```diff
@@ -0,0 +1,91 @@
+# Event basics
+
+## Motivation
+
+The introduction of Events has been contentious, so we want to write down and agree on a few basics.
+
+### What are events?
+
+Events are a type of log which has a required event name and a specific structure which is implied by that event name.
+
+The event name and its implied structure make events much more suitable for observability compared to generic logs.
+
+Because of this, the OpenTelemetry project wants to encourage the transition from generic logs to events.
+
+### OTLP
+
+Since events are a type of log, they share the same OTLP data structure and OTLP pipeline.
+
+### API
+
+OpenTelemetry should have an Event API. This will help to promote the distinction between events and generic logs,
```

Comment:
```suggestion
The OpenTelemetry logger should include methods for emitting events as well as application logs. The term "event" helps to promote the distinction between these two use cases. Methods such as `EmitEvent` should be used for instrumentation, while methods such as `EmitLog` should be used for consuming application logs.
```

Reply:
I agree with Nev that it's very confusing as a customer to use my logger to emit Real User Monitoring (RUM) events. It's a much cleaner story to keep these separate from simple info logs. They are conceptually two very different things. It feels like we're shoehorning in a completely separate concept into an existing construct for some short-term gains. I suggest we stop and ask customers whether they consider a click-event as an enhanced diagnostic log or as a separate concept altogether.

Reply:
Application developers do not really care about these details. I have talked to many developers over the past two years, and they are not bothered by phrases such as "log an event" or "events are stored as logs." If you try to get application developers to dig deeply into this subject they quickly lose interest, it is very clear that it does not stir a lot of passions and they do not really care about the details. 

The only thing that consistently bothered users was the proposal that we not provide an API. When asked to make an event, pretty much all users wanted a line of code to copy and paste. Whether that line of code says `eventLogger.EmitEvent()` or `logger.EmitEvent()` doesn't really matter to them.

When it comes to RUM customers, there is no need for a RUM system to refer to logs. A RUM system also doesn't need to refer to spans, or OpenTelemetry at all. The RUM domain is specific to the browser, so the language used in the RUM UI should match the language used by the browser community to describe browser operations.

The same goes for GenAI. The internal details of OpenTelemetry are not what a specialized GenAI analysis tool designs its UI around. They design using the language of GenAI, aka the semantic conventions. It doesn't really matter what the transport protocol happens to name the data structures, or which API that developers chose to use to generate the data.

Reply:
> Whether that line of code says eventLogger.EmitEvent() or logger.EmitEvent() doesn't really matter to them.

+1. Most people would prefer a helper (possibly auto-generated using weaver/similar) method like `EmitButtonClick(buttonname,username,etc.)`

Reply:
100% agree that the next step in API design for Otel should be to create more of these high level "instrumentor" helpers.

Reply:
Resolving this based on latest updates. If you'd like to continue discussing, please start a new thread with a summary of any remaining points you'd like to discuss. Thanks.